### ISSUE TITLE
Add support for Route `handle`

### ISSUE THREAD

**Issue by PanchoutNathan** [JesusTheHun/storybook-addon-react-router-v6#10]
### **What is the new or updated feature that you are suggesting?**
I upgraded to 6.4.1 in order to use use Matches. However, it has become complicated to use react router dom in storybook to choose path for example. How can I use this new version?

### **Why should this feature be included?**
Be able to use useMatches and createBrowserRouter(router) in storybook decorators and select the routerPath in parameters

**Comment by JesusTheHun** [comment]
Hi, thanks for opening an issue. Can you be more specific about your need ? Ideally showcase your problem with some code !

**Comment by JesusTheHun** [comment]
I took a peak at this new router and its possibilities, there is a lot going on here!
I will try to add basic support in the next couple weeks, I don't have much free time right now.

**Comment by PanchoutNathan** [comment]
> I took a peak at this new router and its possibilities, there is a lot going on here! I will try to add basic support in the next couple weeks, I don't have much free time right now.

Thx a lot ! ;)

**Comment by BenjaminHoch-GoTo** [comment]
Any updates on this? If one of my components makes use of useMatches() the addon throws an error: 
`useMatches must be used within a data router.  See https://reactrouter.com/routers/picking-a-router.`

**Comment by JesusTheHun** [comment]
Yes, I pinned an issue in this repo.
A new version of this addon will be available by the end of next week, with support for data router.

**Comment by JesusTheHun** [comment]
@PanchoutNathan @BenjaminHoch-GoTo new version available, your feedback is greatly appreciated, please check #21

**Comment by BenjaminHoch-GoTo** [comment]
Thank you Jesus, I will try to upgrade today.

**Comment by BenjaminHoch-GoTo** [comment]
so i upgraded to 0.3.1 today but unfortunately for my specific storybook test it doesn't seem to be sufficient. My Breadcrumbs component uses the useMatches() hook from react-router. currently the storybook test uses createMemoryRouter and RouterProvider from react-router-dom to be able to test this.
here are more details on how to use useMatches(): https://reactrouter.com/en/main/hooks/use-matches

in my test I create a mockRouter:
`const createMockRouter = () => {
  return createMemoryRouter(
    [
      {
        path: '/',
        element: <MockLayout />,
        handle: {
          breadcrumbs: [
            { name: 'Breadcrumb 1', to: '/' },
            { name: 'Breadcrumb 2', to: '/' },
          ],
        },
      },
    ],
    { initialEntries: ['/'] },
  )
}`

So basically the functionality to add the handle parameter is missing in your addon.

**Comment by JesusTheHun** [comment]
I've published `v0.3.2` which add support for route handle. Checkout the README.md.

**Comment by BenjaminHoch-GoTo** [comment]
just wanted to report. i am not sure if I am not getting your implementation or if it doesn't work properly. In your examples in the readme you are describing routeHandle and having a string as value. So I tried this
```
export default {
  component: Breadcrumbs,
  decorators: [withRouter],
  parameters: {
    reactRouter: {
      routePath: '/',
      routeHandle: {
        breadcrumbs: [
          { name: 'Breadcrumb 1', to: '/route1' },
          { name: 'Breadcrumb 2', to: '/route2' },
        ],
      },
    },
  },
}
```

and 
```
export default {
  component: Breadcrumbs,
  decorators: [withRouter],
  parameters: {
    reactRouter: {
      routePath: '/',
      routeHandle: "breadcrumbs"
    },
  },
}
```

both are not working. If I `console.log('XXX - useMatches', useMatches())` in my component, the handle from useMatches is still undefined, but should be breadcrumbs..

here is how it should look like (done with createMemoryRouter from react-router-dom:
![image](https://user-images.githubusercontent.com/75682916/226354664-fc880230-7a62-4483-a235-45e6fdad65aa.png)

**Comment by JesusTheHun** [comment]
Ok I've updated all my tests to use the storybook decorator instead of the wrapping component directly.
I have also fixed the issue itself.

Thanks you very much for your follow up & reports 🙏 
Version `0.3.3` is available !

**Comment by BenjaminHoch-GoTo** [comment]
i can confirm it is now working as I expected. Thank you so much for the fast fixes, really appreciate that.

**Comment by JesusTheHun** [comment]
@BenjaminHoch-GoTo  thank you for the kind and positive feedback. It means a lot.
